src/index.html

```diff
@@ -9,8 +9,77 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="images" href="images/Search.svg">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="http://localhost:8000/api"
+      method="post"
+      data-qa="big"
+    >
+
+      <div class="search-box big">
+        <input
+          class="input"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+
+        <button
+          type="submit"
+          class="search-btn big"
```

Comment:
```suggestion
          class="search-btn search-btn_big"
```
if you write some styles for `.big` selector it will be applied to `button` and `div` above
That's not correct behavior

Reply:
i.e instead of using chained selectors add a specific class

Reply:
Done - ready for re-review.